specs/progress.md

```diff
@@ -0,0 +1,150 @@
+- `min`, the minimum value.
+- `max`, the maximum value.
+- `circular`, whether the progress renders circular or not.
+- `paused`, whether the progress is paused or not.
+
+### Prior Art/Examples
+- [FAST-DNA (React)](https://explore.fast.design/components/progress)
+- [Material UI](https://material-ui.com/components/progress/)
+- [Lightning Design / Linear](https://www.lightningdesignsystem.com/components/progress-bar/)
+- [Lightning Design / Circular](https://www.lightningdesignsystem.com/components/progress-ring/)
+- [Ant Design](https://ant.design/components/progress/)
+- [Atlassian / Linear](https://atlaskit.atlassian.com/packages/server/progress-bar)
+- [Atlassian / Circular](https://atlaskit.atlassian.com/packages/core/spinner)
+- [Windows (UWP)](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/progress-controls)
+
+---
+
+### API
+
+*Component name:*
+- `fast-progress`
+
+*Attributes:*
+- `value`: number
+- `min`: number
+- `max`: number
+- `circular`: boolean
+- `paused`: boolean
+
+*Slots:*
+- `indetermimate-linear`
+- `indetermimate-circular`
+
+### Anatomy and Appearance
+
+*Parts:*
+- determinate
+- indeterminate
+- circular-container
+- circular-background
+- circular-determinate
+- circular-indeterminate
+- linear-determinate
+- linear-indeterminate-indicator-1
+- linear-indeterminate-indicator-2
+
+*Circular Determinate Template:*
+```
+<div
+    role="progressbar"
+    aria-valuenow={50}
+    aria-valuemin={0}
+    aria-valuemax={100}
+>
+    <div part="determinate" slot={"determinate"}>
+        <svg part="circular-container" viewBox="0 0 16 16">
+            <circle part="circular-background" cx="8px" cy="8px" r="7px" />
+            <circle part="circular-determinate" style={{stroke-dasharray: "22px 44px"}} cx="8px" cy="8px" r="7px" />
+        </svg>
+    </div>
+</div>
+```
+
+*Circular Indeterminate Template:*
+```
+<div role="progressbar">
+    <div part="indeterminate" slot={"indeterminate"}>
+        <slot name="indetermimate-circular"></slot>
+            <svg part="circular-container" viewBox="0 0 16 16">
+                <circle part="circular-background" cx="8px" cy="8px" r="7px" />
+                <circle part="circular-indeterminate" cx="8px" cy="8px" r="7px" />
+            </svg>
+        </slot>
+    </div>
+</div>
+```
+
+*Linear Determinate Template:*
+```
+<div
+    role="progressbar"
+    aria-valuenow={50}
+    aria-valuemin={0}
+    aria-valuemax={100}
+>
+    <div part="determinate" slot={"determinate"}>
+        <div part="linear-determinate" style={{ width: 50% }}/>
+    </div>
+</div>
+```
+
+*Linear Indeterminate Template:*
+```
+<div role="progressbar">
+    <div part="indeterminate" slot={"indeterminate"}>
+        <slot name="indetermimate-linear">
+            <span part="linear-indeterminate-indicator-1"/>
+            <span part="linear-indeterminate-indicator-2"/>
+        </slot>
+    </div>
+</div>
+```
```

Comment:
Are these all independent?

Reply:
Depending on the properties are passed to the `<Progress/>` component these scenarios would be conditionally rendered. This is simply meant to show the DOM structure of each potential scenario

Reply:
thanks for clarifying!